projects/extension/build.py

```diff
@@ -90,16 +108,68 @@ def idempotent_sql_files() -> list[Path]:
     return paths
 
 
+def parse_feature_flag(path: Path) -> str | None:
+    with path.open(mode="rt", encoding="utf-8") as f:
+        line = f.readline()
```

Comment:
Nit: Would it make sense to skip possible empty lines in the top of the file here?

I believe this should do the job but please double check.


```suggestion
        line = f.readline()
        while line.strip() == '':
            line = f.readline()
```

Reply:
I want to enforce that it be the first line.